addons/mail/static/src/models/follower/follower.js

```diff
@@ -240,6 +241,9 @@ function factory(dependencies) {
         channelName: attr({
             related: 'channel.name',
         }),
+        displayName: attr({
+            related: 'partner.display_name'
+        }),
```

Comment:
This is not correct for a channel follower, it will always be undefined.
It's not making it any worse than before the PR so it's "ok", especially considering channel followers are removed in later version.
But it would have been best to compute this depending on either partner or channel.